sklego/preprocessing/dictmapper.py

```diff
@@ -43,6 +43,41 @@ def fit(self, X, y=None):
         -------
         self : DictMapper
             The fitted transformer.
+
+        Example
```

Comment:
If you manage to add how to make it interact with either [`sklego.preprocessing.ColumnSelector`](https://koaning.github.io/scikit-lego/api/preprocessing/#sklego.preprocessing.pandastransformers.ColumnSelector)  or [`sklearn.composeColumnTransformer`](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.compose) I believe it would be of great help and ready to merge

Reply:
Yes, this double dict was making me really uncomfortable 😅 I'm on it

Reply:
I did the fixes, but I'm unable to push them. I need to figure out what's going on and be back 😅

Reply:
Yes, I was also confused. However, I finally figured out what was happening and was able to push this time. The problem was that one of the files, sklego.model_selection.py, which I wasn’t even working on, failed to pass the ruff-format and was reformatted by ruff. Since I hadn’t worked on it, I decided to revert that change (git restore) and attempted to commit only the two files I had been working on. But after pushing, I received the message “Everything-up-to-date” and didn’t see any changes on my branch. Today, I decided to accept the formatting changes made by ruff, and I was finally able to push. I hope the formatter didn’t break anything in model_selection.py. What’s the proper way to handle this kind of problem in the future?